graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsDataLoaderProvider.kt

```diff
@@ -50,6 +50,7 @@ import kotlin.system.measureTimeMillis
 class DgsDataLoaderProvider(
     private val applicationContext: ApplicationContext,
     private val extensionProviders: List<DataLoaderInstrumentationExtensionProvider> = listOf(),
+    private val scanningInterceptors: List<DgsDataLoaderScanningInterceptor> = listOf(),
```

Comment:
Do we want to support a list of interceptors/customizers? I think it makes sense for instrumentations but not so convinced about opening up to multiple interceptors that can wrap. Especially since it's going to be used internally not something we expose to users?

Reply:
I think it is still useful to expose the ability to add customizers to end users. I see them as a useful building block (or escape hatch?) if our particular instrumentation implementation isn't quite what they want.